<launch>

  <node name="kylinbot_msg_puller"  pkg="kylinbot_core"  type="kylinbot_msg_puller" output="screen">
  <param name="serial_port"         type="string" value="/dev/ttyTHS2"/>  
  <param name="serial_baudrate"     type="int"    value="115200"/>
  <param name="spin_rate"           type="int"    value="100"/>
  </node>

  <node name="kylinbot_msg_pusher"  pkg="kylinbot_core"  type="kylinbot_msg_pusher" output="screen">
  <param name="serial_port"         type="string" value="/dev/ttyTHS2"/>  
  <param name="serial_baudrate"     type="int"    value="115200"/>
  <param name="spin_rate"           type="int"    value="100"/>
  </node>
<!--
  <node name="kylinbot_video_cap"   pkg="kylinbot_core"  type="kylinbot_video_cap" output="screen">
  <param name="device"              type="string" value="/dev/video0"/>
  <param name="showimg"             type="bool"   value="false"/>
  <param name="winname"             type="string" value="video_cap"/>
  <param name="buf_size"            type="int"    value="3"/>
  <param name="exp_time"            type="int"    value="62"/>
  <param name="gain"                type="int"    value="30"/>
  <param name="brightness"          type="int"    value="10"/>
  <param name="whiteness"           type="int"    value="86"/>
  <param name="saturation"          type="int"    value="60"/>
  <param name="spin_rate"           type="int"    value="100"/>
  </node>

  <node name="kylinbot_image_pro"  pkg="kylinbot_core"  type="kylinbot_image_pro" output="screen">
  <param name="spin_rate"           type="int"    value="100"/>
  </node>
 --> 
  <node name="kylinbot_logic_controller"  pkg="kylinbot_core"  type="kylinbot_logic_controller" output="screen">
  <param name="spin_rate"           type="int"    value="100"/>
  </node>

</launch>
